test/e2e_flags.go

```diff
@@ -42,13 +42,15 @@ var (
 type EnvironmentFlags struct {
 	env.ClientConfig
 	testenv.TestClientConfig
+	env.ControllerConfig
```

Comment:
These test flags shouldn't be used in any controllers so you should be able to drop the changes in this file

Reply:
done